typing/ctype.ml

```diff
@@ -2571,16 +2567,13 @@ let unify_package env unify_list lv1 p1 fl1 lv2 p2 fl2 =
 
 
 (* force unification in Reither when one side has a non-conjunctive type *)
+(* TODO: this could also be put in unification_environment? *)
```

Comment:
Looks to me that this would go somewhere else -- maybe just a (labeled) boolean that gets passed around the `eqtype` functions? Those functions don't otherwise need the new env type available, so it seems that putting `rigid_variants` there would be a poor fit.`

Reply:
Is this still TODO?